WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -5201,7 +5198,6 @@
 				E157D5E01C690A6C00F04FB9 /* ImmuTableTestUtils.swift in Sources */,
 				E1C9AA561C10427100732665 /* MathTest.swift in Sources */,
 				93A379EC19FFBF7900415023 /* KeychainTest.m in Sources */,
-				17C1BA0E1C68D63C0017F38C /* PlanDetailViewControllerTest.swift in Sources */,
```

Comment:
`PlanDetailViewControllerTest.swift` was removed from the project but the file is still in the repo